### DISCUSSION TITLE
Issue with "Error: School email must be verified" on GitHub Student Pack

### DISCUSSION THREAD

**Discussion by RoTaDu2899** [community/community#147892] [GitHub Education]
### Hi everyone,

I’m encountering an issue while trying to apply for the GitHub Student Developer Pack. I’ve successfully verified my school email address on my GitHub account, and I’ve also set it as my primary email. However, I still receive the following error message when I attempt to apply:

"Error: School email must be verified on your GitHub account."

**Comment by davevad93** [comment]
Hi @RoTaDu2899 ,

If you have successfully verified your academic email and set it as primary and are still getting this error, try logging out and logging back in from GitHub and the GitHub Education portal; if the problem persists, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.
